octopus-agent/src/main/java/octopus/teamcity/agent/OctopusBuildInformationBuildProcess.java

```diff
@@ -63,7 +63,7 @@ protected OctopusCommandBuilder createCommand() {
                     sharedConfigParameters.get("octopus_vcstype"),
                     sharedConfigParameters.get("octopus_vcsroot"),
                     sharedConfigParameters.get("build.vcs.number"),
-                    sharedConfigParameters.get("octopus_branch"),
+                    sharedConfigParameters.get("vcsroot.branch"),
```

Comment:
This feels like we didn't assign `vcsroot.branch` to `octopus_branch` parameter properly somewhere